app/presenters/download_options/audio_download_options.rb

```diff
@@ -15,6 +15,14 @@ def options
       # We don't use content_type in derivative option subheads,
       # cause it's in the main label. But do use it for original.
 
+      if m4a_deriv = asset.file_derivatives[:m4a]
+        options << DownloadOption.with_formatted_subhead("M4A",
```

Comment:
How do we want to label this download?

Reply:
How does it come out with what you are currently doing?  What do you think?

Reply:
Currently just "M4A". I'm fine with that.
![Screen Shot 2022-08-01 at 10 33 05 AM](https://user-images.githubusercontent.com/94723/182173572-548b2c01-a12d-450a-9c35-7da1da1a077a.png)
.

Reply:
I suppose we could put something in the sub-head like "smaller file", I dunno. Or "Smaller file (M4A -- 21.9 MB)" vs "Original file (FLAC -- 416MB)".  Just brainstorming. Not sure if everyone knows what M4A is -- but it's really what they ought to be downloading unless they specifically want the FLAC!

Reply:
Good call